x-pack/plugins/security_solution/server/endpoint/routes/actions/service.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger } from 'kibana/server';
+import type { estypes } from '@elastic/elasticsearch';
+import { AGENT_ACTIONS_INDEX, AGENT_ACTIONS_RESULTS_INDEX } from '../../../../../fleet/common';
+import { SecuritySolutionRequestHandlerContext } from '../../../types';
+
+export const getAuditLogESQuery = ({
+  elasticAgentId,
+  from,
+  size,
+}: {
+  elasticAgentId: string;
+  from: number;
+  size: number;
+}): estypes.SearchRequest => {
+  return {
+    index: [AGENT_ACTIONS_INDEX, AGENT_ACTIONS_RESULTS_INDEX],
+    size,
+    from,
+    body: {
+      query: {
+        bool: {
+          should: [
+            { terms: { agents: [elasticAgentId] } },
+            { terms: { agent_id: [elasticAgentId] } },
+          ],
+        },
+      },
+      sort: [
+        {
+          '@timestamp': {
+            order: 'desc',
+          },
+        },
+      ],
+    },
+  };
+};
+
+export const getAuditLogResponse = async ({
```

Comment:
Can you add a response Type to this function

Reply:
done 04347187d9718ca61ccd0a53fa32452b6a7d481e